app/docs/setup_code.tsx

```diff
@@ -175,126 +187,194 @@ export default class SetupCodeComponent extends React.Component {
     event.target.innerText = "Copied!";
   }
 
+  onChangeCredential(e: React.ChangeEvent<HTMLSelectElement>) {
+    this.setState({ selectedCredentialIndex: Number(e.target.value) });
+  }
+
+  onClickLink(href: string, e: React.MouseEvent<HTMLAnchorElement>) {
+    e.preventDefault();
+    router.navigateTo(href);
+  }
+
+  renderMissingApiKeysNotice() {
+    return (
+      <div className="no-api-keys">
+        <div className="no-api-keys-content">
+          <div>
+            {capabilities.anonymous ? (
+              <>Looks like your organization doesn't have any API keys.</>
+            ) : (
+              <>This BuildBuddy installation requires API key authentication, but no API keys are set up.</>
```

Comment:
this is a little confusing because in the users eyes, they might be using certs

maybe something a little simpler like
```
This BuildBuddy installation requires authentication, but no API keys are set up.
```

Reply:
Ah good point, fixed